src/Sulu/Bundle/WebsiteBundle/Navigation/NavigationMapperInterface.php

```diff
@@ -57,7 +58,8 @@ public function getRootNavigation(
         $depth = 1,
         $flat = false,
         $context = null,
-        $loadExcerpt = false
+        $loadExcerpt = false,
+        $segmentKey = null
```

Comment:
Was this committed accidentally? Seems this change  has nothing todo with this PR?